### PR TITLE
Feat add favicon

### PR DESCRIPTION
**Pull request by Dukent2903**
I changed previous favicon and put a start ui icon 

related to #113

### CODE REVIEW

File: public/site.webmanifest [hunk 1]
```diff
@@ -1,19 +1,14 @@
 {
-    "name": "Start UI",
-    "short_name": "Start UI",
+    "name": "start UI",
+    "short_name": "start UI",
```
Comment (yoannfleurydev) [inline public/site.webmanifest:3]: Please keep the case

```suggestion
    "name": "Start UI",
    "short_name": "Start UI",
```

**Comment by ivan-dalmet** [comment]
Some image have background 😕
image
All image should have transparent backgrounds

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]
Thanks! 

But some images have background 😕
image
image

All image should have transparent backgrounds

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -83,16 +83,9 @@ const AppHead = () => {
...
-        color={theme.colors.gray?.['800']}
-      />
-      <meta
-        name="msapplication-TileColor"
-        content={theme.colors.gray?.['800']}
-      />
-      <meta name="theme-color" content={theme.colors.gray?.['800']} />
+      <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+      <meta name="msapplication-TileColor" content="#ffc40d" />
+      <meta name="theme-color" content="#ffffff" />
```
[the remark this answers is not in the document]
Comment (Dukent2903) [reply src/pages/_app.tsx:88]: I'm not much sure about hardcoded strings but does it mean to ramain with the same syntax?

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ const AppHead = () => {
       <link
         rel="apple-touch-icon"
         sizes="180x180"
-        href="/apple-touch-icon.png"
+        href="//apple-touch-icon.png"
```
[the remark this answers is not in the document]
Comment (Dukent2903) [reply src/pages/_app.tsx:71]: smae as here I geuss??

File: public/site.webmanifest [hunk 4]
```diff
@@ -6,14 +6,9 @@
...
-        {
-            "src": "/android-chrome-384x384.png",
-            "sizes": "384x384",
-            "type": "image/png"
         }
     ],
-    "theme_color": "#000000",
-    "background_color": "#000000",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
```
Comment (yoannfleurydev) [inline public/site.webmanifest:12]: not sure we want to update that part

Comment (Dukent2903) [reply public/site.webmanifest:12]: Sorry about the late respond, I will correct them

**Review by yoannfleurydev — APPROVED** [review]

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]
Still errors in images (red background)
image

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]
Ivan is right, their are still background on images.

image

File: public/open-api.json [hunk 5]
```diff
@@ -1,524 +0,0 @@
-{
```
Comment (yoannfleurydev) [inline public/open-api.json:1]: This file shouldn't be removed.

Comment (Dukent2903) [reply public/open-api.json:1]: ok working on it

**Review by yoannfleurydev — APPROVED** [review]

**Comment by yoannfleurydev** [comment]
Updated the file and squashed commits. Still a question though, https://realfavicongenerator.net/ gives an advice about the icon being transparent and that may look weird on iOS

image

That is a great advice so I did add a white background (black background was weird) and some padding to the icon

image

But, the thing is, checking the preview with an iOS device using Safari, this looks kinda weird as it is not the behaviour announced by Real Favicon Generator

image

**Comment by yoannfleurydev** [comment]
Closing this PR, will do a cleaner one.

**Pull request closed without merging**